docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.num_samples                 | integer         | Number of samples to use from the dev set, -1 mean all                                                       |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.batch_size                  | integer         | Dev set batch size                                                                                           |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.name                                | string          | Optimizer to use for training                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.lr                                  | float           | Learning rate to use for training                                                                            |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.weight_decay                        | float           | Weight decay to use for training                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.name                          | string          | Warm up schedule                                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | Warm up ratio                                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+To train the model from scratch, run:
+
+.. code::
+
+      python examples/nlp/token_classification/punctuation_and_capitalization_train.py \
+             model.dataset.data_dir=<PATH/TO/DATA_DIR> \
+             trainer.gpus=[0,1] \
+             optim.name=adam \
+             optim.lr=0.0001 \
+             model.nemo_path=<PATH/TO/SAVE/.nemo>
+
```

Comment:
How about explaining what it does? something like:"It trains the model on 2 gpus 0 and 1 with adam optimzier  and learnign rate of 0.0001. And stores the final model is the path specifie dby model.nemo_path..."